src/OrchardCore/OrchardCore.DisplayManagement/Implementation/DefaultHtmlDisplay.cs

```diff
@@ -257,6 +265,17 @@ static IHtmlContent CoerceHtmlString(object value)
             return new HtmlString(HtmlEncoder.Default.Encode(value.ToString()));
         }
 
+        private static bool TryGetParentShapeTypeName(ref string shapeTypeScan)
```

Comment:
What's the point of the `ref` argument? strings are already passed by ref.

Reply:
It's to assign the parent shape type name to `shapeTypeScan`.

Reply:
Not a pattern we use a lot, but I understand now. We usually prefer `out` arguments in this case. You choose.

Reply:
I think I never used it before, but `out` switch dont requires the parameter to be assigned and requires to assign it in all cases. So `ref` is the most suitable.